### PR TITLE
Bugfix #210 model not saved after inserting images

### PR DESCRIPTION
**Pull request by rbecheras**

### CODE REVIEW

File: src/tinymce.js [hunk 1]
```diff
@@ -80,6 +80,16 @@ angular.module('ui.tinymce', [])
               updateView(ed);
             });
 
+          // Update model on Set content
+          // Only solution I found to fix #210
```
Comment (deeg) [inline src/tinymce.js]: Can you remove lines 84 & 85 please? They are not necessary if merging this in.

File: src/tinymce.js [hunk 2]
```diff
@@ -80,6 +80,16 @@ angular.module('ui.tinymce', [])
...
+          // https://github.com/angular-ui/ui-tinymce/issues/210
+          ed.on('SetContent', function(evt) {
+            if (evt.content) {
```
Comment (deeg) [inline src/tinymce.js]: Could SetContent ever get called with an empty string and still want to update? I believe so. Instead should we just update [this block|https://github.com/angular-ui/ui-tinymce/blob/master/src/tinymce.js#L77-L81] to include SetContent?

**Comment by deeg** [comment]
I know the tests are a bit out of date, and I have a task to clean them up and add more, but it would be nice if you could add tests for this as part of the PR.

Once the changes are made we can get this merged in. Thanks for reporting and posting a PR.

**Comment by rbecheras** [comment]
Very well, I should do it this week. Thanks

**Comment by rbecheras** [comment]
@deeg , sorry I missed to update the code for a while, but it's done.
You can merge it now.

**Comment by deeg** [comment]
Please see my comment on #210 about explaining why SetContent is needed.

**Comment by deeg** [comment]
@rbecheras, going to close this PR. Lets finish the discussion over at #210, and we can get another PR opened pending the results.

**Pull request closed without merging**